src/Nancy/DynamicDictionaryValue.cs

```diff
@@ -333,11 +347,19 @@ public override bool TryConvert(ConvertBinder binder, out object result)
             return base.TryConvert(binder, out result);
         }
 
+        /// <summary>
+        /// Returns the string representation for a DynamicDictionaryValue instance

```

Comment:
All mentions of `DynamicDictionaryValue` in this class (there are several in the XML comments below) should be `<see cref="DynamicDictionaryValue"/>`